### PR TITLE
KAFKA-17526: make ConfigCommandIntegrationTest.java test use correct arguments in testing alias

### PR DESCRIPTION
**Pull request by frankvicky**
JIRA: [KAFKA-17526](https://issues.apache.org/jira/browse/KAFKA-17526)

This PR reviews existing tests and refactors the validation to ensure the command is generated correctly. 

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java [hunk 1]
```diff
@@ -324,11 +324,13 @@ public void testDynamicBrokerConfigUpdateUsingKraft() throws Exception {
 
     @ClusterTest(types = {Type.KRAFT, Type.CO_KRAFT})
     public void testGroupConfigUpdateUsingKraft() throws Exception {
-        alterOpts = asList("--bootstrap-server", cluster.bootstrapServers(), "--entity-type", "groups", "--alter");
+        alterOpts = Stream.concat(entityOp(Optional.of(defaultGroupName)).stream(),
```
Comment (chia7712) [inline core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java]: Can we make alterOpts a local variable? it is a bit chaos that `alterOpts` can be modified by any method.

**Review by chia7712 — COMMENTED** [review]
@frankvicky thanks for your refactor

**Comment by chia7712** [comment]
@frankvicky Could you please rebase code to include recent fix?

**Comment by chia7712** [comment]
@frankvicky please fix the conflicts

**Comment by frankvicky** [comment]
Hi @chia7712 
I have rebased and fixed the conflicts, PTAL 😺

**Review by chia7712 — APPROVED** [review]

**Pull request merged**